src/scripts/main.js

```diff
@@ -1 +1,66 @@
 'use strict';
+
+let rightButtonClicked = false;
+let leftButtonClicked = false;
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    leftButtonClicked = true;
+    resolve('First promise was resolved');
+  });
+
+  if (!leftButtonClicked) {
+    setTimeout(() => {
+      // eslint-disable-next-line
+      reject(`First promise was rejected`);
+    }, 3000);
+  }
```

Comment:
The rejection logic for the first promise will not work as intended because the promise is already resolved on the first click. Consider restructuring the logic to handle the rejection condition more effectively.